dotCMS/src/main/java/com/dotmarketing/cms/myaccount/action/MyAccountAction.java

```diff
@@ -248,7 +248,7 @@ private void loadUserInfoInRequest(MyAccountForm form, String userId,
 	public static boolean subscribeDotCMSMailingList(User user) {
 		String to = "dotcms-subscribe@yahoogroups.com";
 		String from = user.getEmailAddress();
-		String subject = "Subscribe to dotCMS mailing list";
```

Comment:
@wezell ah this looks like an unwanted change from Sanchez's local got into the PR